src/test/simple_spin.cc

```diff
@@ -46,7 +46,8 @@ TEST(SimpleSpin, Test1)
   // Should also work with pass-by-reference:
   // (Note that we don't care about cross-threading here as-such.)
   counter = 0;
-  ASSERT_EQ(counter, 0);
+  uint32_t urhs = 0;
+  ASSERT_EQ(counter, urhs);
```

Comment:
0u seems good. Kefu is correct in that it's not necessary (we do expect assignment to an integer to work, or we have bigger problems), but I personally prefer it there for the reason I added it: it is explicitly checking the precondition, which makes me feel warm and fuzzy. That all said, I don't want to make a mountain out of a molehill, so fix as you see fit (and, apologies that I didn't notice the warning myself!).